docs/e2e_test_users.md

```diff
@@ -0,0 +1,28 @@
+# E2E Test users
+
+# Define a test user
+
+Go to the admin interface to `/admin/e2e_tests/testuser/` to define a test user.
+
+# Generate a token
+
+Using the admin graphql endpoint execute the follwing query:
+
+GraphQL mutation:
+```graphql
+mutation {
+  generateTestUserToken(input: { userEmail: "[test user email]"}) {
+    errors
+    token
+  }
+}
+```
+
+Curl command:
+```bash
+curl '[host]/graphql/admin' \
```

Comment:
suggestion: can we define a shell script or python manage.py task to do this?

Reply:
@paulschreiber That depends, without the HOST and the session ID explicitley defined, no, that is not possible, a script that takes those 2 as parametes is possible, but ideally the intent of this is to execute an admin operation remotely, not connection to the host directly